std-bits/base/src/main/java/org/enso/base/time/Date_Period_Utils.java

```diff
@@ -0,0 +1,21 @@
+package org.enso.base.time;
+
+import java.time.temporal.ChronoField;
+import java.time.temporal.Temporal;
+import java.time.temporal.TemporalAdjusters;
+
+public class Date_Period_Utils implements TimeUtilsBase {
```

Comment:
@jdunkerley do you think I should add some doc comments? Seems a bit 'naked' without them but OTOH I'd say the name of the function seems to be pretty self explanatory so I'm not sure if the comments would add any clarity. What do you think?